iron-form.html

```diff
@@ -135,27 +158,25 @@
       */
 
       attached: function() {
-        this._nodeObserver = Polymer.dom(this.$.content).observeNodes(
```

Comment:
I don't think I broke it and I think there are tests for this case. It's now using `slotchange` event. It will recursively find all slots and subscribe to them. See `_getFormElements`.